lib/translator.js

```diff
@@ -11,9 +11,11 @@ var TranslatedFile = require('./translated-file');
 var PluginError = gutil.PluginError;
 
 
+
 function Translator(options) {
   this.options = lodash.defaults(options, {
       localeDir: 'locale',
+      msgDirs: ['locale'],
```

Comment:
Two options doesn't make sense, I would unify on `localeDirs`.

Normally I would be super cautious about altering the options (especially on a public repo) but the plugin is still ~v000 and not yet implemented anywhere so I feel it's appropriate to make significant changes as this thing find's it's stride.